src/parsers.php

```diff
@@ -475,10 +477,8 @@ function isFallible() : bool
     };
 }
 
-const
-    CONSUME_DO_TRIM = 0x10,
-    CONSUME_NO_TRIM = 0x01
-;
+if(!defined(__NAMESPACE__."\\CONSUME_DO_TRIM")) define(__NAMESPACE__."\\CONSUME_DO_TRIM", 0x10);
+if(!defined(__NAMESPACE__."\\CONSUME_NO_TRIM")) define(__NAMESPACE__."\\CONSUME_NO_TRIM", 0x01);
```

Comment:
Hi,

Can't we put the `if(!defined(__NAMESPACE__."\\LAYER_DELIMITERS"))` conditions on the IDE stub files and keep the source as it is? These conditions make the source look polluted.